config.json

```diff
@@ -1,7 +1,7 @@
 {
   "language": "Emacs Lisp",
   "active": true,
-  "blurb": "Emacs Lisp is a variant of lisp used as a scripting language by the Emacs text edior. Not only is it used to customize Emacs, but much of the core editing functionality itself is written in Emacs Lisp",
+  "blurb": "Emacs Lisp is the extension language of the Emacs text editor and as such one of the most used Lisp dialects in the world. It is used to write full-fledged applications that run within Emacs but also user preference files.",
```

Comment:
Maybe we can remove the preference files reference altogether :)

Reply:
What about this →

... It is used to write personal extensions to Emacs, which are preference files on steroids, but also full-fledged applications that run within Emacs itself.

Reply:
I'd like to lean away from the "on steroids" language. I'm not sure if it would be problematic, but my general rule is that if I have to ask, then it might end up that way.

Reply:
Ok, let me start from semi-scratch:

Emacs Lisp is the extension language of the Emacs text editor. You can easily create you own extensions, which include preference files, but also full-fledged applications that use Emacs as a running environment in a fully integrated way.

Reply:
I like it! 👍